packages/block-library/src/embed/edit.js

```diff
@@ -84,7 +84,10 @@ const EmbedEdit = ( props ) => {
 				return { fetching: false, cannotEmbed: false };
 			}
 
-			const embedPreview = getEmbedPreview( attributesUrl );
+			const embedPreview = Platform.select( {
```

Comment:
The idea here was to mock the embed preview to imitate the same flow as it would do after fetching a real preview. The important part is to generate a valid value for populating the `preview` variable, otherwise, the embed preview component is not rendered.

https://github.com/WordPress/gutenberg/blob/de3b53fb66fbd188f07c46a3c824371f6d1d3107/packages/block-library/src/embed/edit.js#L98-L101
https://github.com/WordPress/gutenberg/blob/de3b53fb66fbd188f07c46a3c824371f6d1d3107/packages/block-library/src/embed/edit.js#L216-L217

Reply:
Ah, I see 👍